espanol/canudos.md

```diff
@@ -115,3 +115,5 @@ Lo que nadie sabía, es que el consejero era un versado maestro instruído en la
 No tuvo más opción que buscar refuerzos, luchadores incanzables que tuvieran las siguientes características:
 
 - [Características](caracteristicas/caracteristicas.md)
+
+Porque en el fondo él sabía que por mucho que hubiese ganado esta batalla, la guerra no había hecho más que [comenzar](guerra/duracion.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.